build/source/engine/vegNrgFlux.f90

```diff
@@ -2070,104 +1760,92 @@ subroutine aeroResist(&
+ real(rkind)                      :: canopyLeafConductance                ! leaf boundary layer conductance -- scaled up to the canopy (m s-1)
+ real(rkind)                      :: leaf2CanopyScaleFactor               ! factor to scale from the leaf to the canopy [m s-(1/2)]
  ! -----------------------------------------------------------------------------------------------------------------------------------------
- ! initialize error control
- err=0; message='aeroResist/'
+ err=0; message='aeroResist/' ! initialize error control
 
- ! check that measurement height is above the top of the canopy
- if(mHeight < heightCanopyTop)then
+ if (mHeight < heightCanopyTop) then ! check that measurement height is above the top of the canopy
   err=20; message=trim(message)//'measurement height is below the top of the canopy'; return
  end if
 
- ! -----------------------------------------------------------------------------------------------------------------------------------------
  ! * compute vegetation poperties (could be done at the same time as phenology.. does not have to be in the flux routine!)
- if(computeVegFlux) then ! (if vegetation is exposed)
-
+ if (computeVegFlux) then ! if vegetation is exposed
   ! ***** identify zero plane displacement, roughness length, and surface temperature for the canopy (m)
   ! First, calculate new coordinate system above snow - use these to scale wind profiles and resistances
   ! NOTE: the new coordinate system makes zeroPlaneDisplacement and z0Canopy consistent
   heightCanopyTopAboveSnow = heightCanopyTop - snowDepth
   heightCanopyBottomAboveSnow = max(heightCanopyBottom - snowDepth, 0.0_rkind)
   select case(ixVegTraits)
-
    ! Raupach (BLM 1994) "Simplified expressions..."
    case(Raupach_BLM1994)
-    ! (compute zero-plane displacement)
+    ! compute zero-plane displacement
     funcLAI          = sqrt(c_d1*exposedVAI)
     fracCanopyHeight = -(1._rkind - exp(-funcLAI))/funcLAI + 1._rkind
     zeroPlaneDisplacement = fracCanopyHeight*(heightCanopyTopAboveSnow-heightCanopyBottomAboveSnow)+heightCanopyBottomAboveSnow
-    ! (coupute roughness length of the veg canopy)
-    approxDragCoef   = min( sqrt(C_s + C_r*exposedVAI/2._rkind), approxDragCoef_max)
+    ! coupute roughness length of the veg canopy
+    approxDragCoef   = min(sqrt(C_s + C_r*exposedVAI/2._rkind), approxDragCoef_max)
     z0Canopy         = (1._rkind - fracCanopyHeight) * exp(-vkc*approxDragCoef - psi_h) * (heightCanopyTopAboveSnow-heightCanopyBottomAboveSnow)
-
    ! Choudhury and Monteith (QJRMS 1988) "A four layer model for the heat budget..."
    case(CM_QJRMS1988)
     funcLAI =  cd_CM*exposedVAI
-    zeroPlaneDisplacement = 1.1_rkind*heightCanopyTopAboveSnow*log(1._rkind + funcLAI**0.25_rkind)
+    zeroPlaneDisplacement = 1.1_rkind*heightCanopyTopAboveSnow*log(1._rkind + sqrt(sqrt(funcLAI))) ! using sqrt intrinsic for speed
```

Comment:
This new comment only makes sense in the context of what was changed. Can this be something like:
```
Original equation  1.1_rkind*heightCanopyTopAboveSnow*log(1._rkind + funcLAI**0.25_rkind) implemented using sqrt() for speed
```

Reply:
I have added the original equation in the comment for context.